### PR TITLE
Conversion of note Durations to decimals/quantization/smoothing

### PR DESCRIPTION
**Pull request by anstepp**

### CODE REVIEW

File: tests/test_transcribe.py [hunk 1]
```diff
@@ -99,23 +100,51 @@ def test_smooth_notes(load_sample_audio):
 
     f0_range = (24, 48)
 
+    print("***")
```
Comment (rachhouse) [inline tests/test_transcribe.py:103]: Generally you don't want to leave `print` statements in your code - they litter. Log debug is a good alternative if you need to have some sort of output.

**Review by rachhouse — COMMENTED** [review]
Looks good, just a few small things to address.

File: lejaren/analysis/AutoTranscribe.py [hunk 2]
```diff
@@ -174,7 +177,7 @@ def _transform_x(
             Xr = rfft(xzerophase)
 
             # Get amplitude of bins lower than 800.
-            Xrmag = abs(Xr)[:1200]
+            Xrmag = abs(Xr)[0:1200]
```
[the remark this answers is not in the document]
Comment (anstepp) [reply lejaren/analysis/AutoTranscribe.py]: Thanks.

File: lejaren/analysis/AutoTranscribe.py [hunk 3]
```diff
@@ -389,13 +392,48 @@ def _ideal_to_measured_mismatch(self, peaks, ideal_spectrum):
...
+                note_list[idx-1].change_duration(note.dur + note_list[idx-1].dur)
+                note_list.pop(idx)
+
+        added_list = []
+
         for idx, note in enumerate(note_list):
-            if note.dur == one_frame_dur:
-                one_frame_indices.append(idx)
+            if idx >= 1:
+                if note.pc == note_list[idx-1].pc and note.octave == note_list[idx-1].octave:
```
[the remark this answers is not in the document]
Comment (anstepp) [reply lejaren/analysis/AutoTranscribe.py:409]: Thanks.

File: lejaren/analysis/AutoTranscribe.py [hunk 4]
```diff
@@ -389,13 +392,48 @@ def _ideal_to_measured_mismatch(self, peaks, ideal_spectrum):
...
+                    added_list[-1].change_duration(note.dur + note_list[idx-1].dur)
+                else:
+                    added_list.append(note)
+            else:
+                added_list.append(note)
+
+        quantized_notes = self.quantize_notes(added_list, 0.125)
```
[the remark this answers is not in the document]
Comment (anstepp) [reply lejaren/analysis/AutoTranscribe.py]: I did it!

**Review by rachhouse — APPROVED** [review]
lgtm

**Pull request merged**